test/test_utils.py

```diff
@@ -60,3 +65,51 @@ def test_get_binary_env_var_wrong_value():
     finally:
         if key in os.environ:
             del os.environ[key]
+
+
+# To test from+ range and that this function is correctly selected as the implementation.
+@implement_for("_utils_internal", "0.3")
```

Comment:
Can we wrap those tests under a `TestVersioning` class or something like that?

Reply:
Done with `implement_for_test_functions` class.